Description: Fix manpage
 This patches fixes the manpage installation for Debian package only
Author: Jonathan Bennett <jbennett@cipherdyne.com>
Forwarded: no

---
Index: fwknop-gui/CMakeLists.txt
===================================================================
--- fwknop-gui.orig/CMakeLists.txt
+++ fwknop-gui/CMakeLists.txt
@@ -87,7 +87,7 @@ add_custom_target(man ALL DEPENDS ${MAN_
   install(TARGETS fwknop-gui DESTINATION bin)
   install(PROGRAMS fwknop-gui.desktop DESTINATION share/applications)
   install(FILES fwknop-gui.png DESTINATION share/pixmaps)
-  INSTALL(FILES fwknop-gui.8 DESTINATION share/man/man8)
+  INSTALL(FILES ${CMAKE_BINARY_DIR}/fwknop-gui.8 DESTINATION share/man/man8)
 endif()
 if(IS_WIN32)
   include_directories( ${LIBQRENCODE_INCLUDE_DIR} )
